pkg/storaged/details.js

```diff
@@ -34,7 +34,25 @@ define([
      */
 
     function init_details(client, jobs) {
-        var type, name;
+        var type, name, state;
+
+        function ensure_state_multipathd_running() {
+            if (state.multipathd_running === undefined) {
```

Comment:
If starting the service produces a good configuration, Cockpt will pick that up and show it.

If starting the service doesn't change the configuration of the drive, nothing happens in the UI.